site2/website-next/docusaurus.config.js

```diff
@@ -254,28 +205,10 @@ module.exports = {
           position: "right",
         },
         {
-          label: "Version",
-          to: "/docs",
-          position: "right",
-          items: [
-            {
-              label: "2.9.0",
-              to: "/docs/2.9.0/",
-            },
-            {
-              label: "2.8.0",
-              to: "/docs/2.8.0/",
-            },
-            {
-              label: "2.7.0",
-              to: "/docs/2.7.0/",
-            },
-            {
-              label: "2.6.0",
-              to: "/docs/2.6.0/",
-            },
-          ],
-        },
```

Comment:
Why remove these?

Are these your aligned requests? Do I miss anything? @D-2-Ed @DaveDuggins

Reply:
Talked w/ @ @D-2-Ed, "Version" should be replaced w/ "Download" (or other word). The "Download" page should contain [new "Release notes" page](https://docs.google.com/spreadsheets/d/1MT8vt0620Cy4tggKk3gDVPbnhH7GVakld-5aDYCGAko/edit#gid=149553782).